prow/plugins/trigger/ic.go

```diff
@@ -57,13 +58,14 @@ func handleIC(c client, trustedOrg string, ic github.IssueCommentEvent) error {
 	}
 
 	// Which jobs does the comment want us to run?
-	testAll := okToTest.MatchString(ic.Comment.Body)
-	shouldRetestFailed := retest.MatchString(ic.Comment.Body)
+	okToTest := okToTestRe.MatchString(ic.Comment.Body)
+	testAll := okToTest || testAllRe.MatchString(ic.Comment.Body)
+	shouldRetestFailed := retestRe.MatchString(ic.Comment.Body)
 	requestedJobs := c.Config.MatchingPresubmits(ic.Repo.FullName, ic.Comment.Body, testAll)
 	if !shouldRetestFailed && len(requestedJobs) == 0 {
 		// Check for the presence of the needs-ok-to-test label and remove it
 		// if a trusted member has commented "/ok-to-test".
-		if testAll && ic.Issue.HasLabel(needsOkToTest) {
+		if okToTest && ic.Issue.HasLabel(needsOkToTest) {
```

Comment:
is this comment wrong or has this been broken for a while?

Reply:
I don't follow, the comment looks correct to me and I don't see why this would have been broken.
Note that the `testAll` variable has changed meaning in this PR, previously it was true iff the comment was an `/ok-to-test` command.

Reply:
I see, the old `testAll` is somewhat confusingly named. :+1: